pkg/scene/scan.go

```diff
@@ -333,3 +357,48 @@ func getInteractive(path string) bool {
 	_, err := os.Stat(GetFunscriptPath(path))
 	return err == nil
 }
+
+func getCaption(path string) string {
+	captions := ""
+	parent := filepath.Dir(path)
+	baseName := filepath.Base(path)
+
+	// only proceed if we haven't already looked at this directory
+	if previousDir != parent {
```

Comment:
This limits us to having subtitles for a single scene per folder. We cant have more than 1 scene with subtitles in the same folder or not?

Reply:
It does limit us to having subtitles for a single scene per folder. It also prevents us from going through a folder with 20 scenes several times. As we expand the number of support files Stash supports for a scene, it makes sense to have a file structure that contains 1 scene per folder.